doc/rtd/topics/code_review.rst

```diff
@@ -0,0 +1,170 @@
+*******************
+Code Review Process
+*******************
+
+In order to manage incoming pull requests effectively, and to ensure
+that community members (like you!) submitting them understand what to
+expect, cloud-init has a documented code review process.
+
+The language used is intentionally formal, in a bid to avoid ambiguity,
+but the *experience* of working through the code review process should
+not be, so please don't be put off!  If you have any questions about
+this process, please do ask them in ``#cloud-init`` on the Freenode IRC
+network.
+
+Goals
+=====
+
+This process has the following goals:
+
+* Ensure that cloud-init community members are receiving code reviews
+  in a timely fashion
+* Minimize the time the cloud-init core developers have to spend on the
+  parts of the code review process which aren't actual code review
+
+Roles
+=====
+
+For ease of understanding, we will refer to two different roles
+throughout the process:
+
+Proposer
+   The developer proposing a pull request (hereafter known as a PR).
+   This could be a core developer or community member.  (This is likely
+   the role you will be taking in this process.)
+
+the Committers
+   The group of cloud-init core developers.
+
+   In this process doc, cloud-init core developers are treated as a
+   bloc of interchangeable people.  This is a simplification, as in
+   reality we will need to manage transfer of reviews, dismissal of
+   stale reviews, parallel reviews from multiple committers, etc.  If
+   we find that handling these on an ad-hoc basis is causing problems,
+   then we should revisit and expand this process to include them.
+
+   (If the **Proposer** is a core developer, then they are not
+   considered part of this group for the purposes of this process.)
+
+Opening Phase
+=============
+
+In this phase, the **Proposer** is responsible for all actions. They
+should get a PR into good enough shape that it is worth **the
+Committers** spending time reviewing it.  Specifically, they are
+responsible for getting to a point where the continuous integration
+(CI) testing in Travis is passing.  Once a PR is passing CI, it moves
+into the :ref:`Review phase`.
+
+These are the steps that comprise the opening phase:
+
+1. The **Proposer** opens PR
+
+2. An automated comment will be added to the PR, outlining the steps
+   expected of the **Proposer** before a **Committer** will review
+   their PR.  (N.B. This automation is not yet implemented!)
+
```

Comment:
As stated above I think there are different classes of "requests for reviews" I think those will need to be taken into account in any automation.